zion/src/test/scala/edu/uci/ics/cloudberry/zion/actor/DataStoreManagerTest.scala

```diff
@@ -291,13 +281,80 @@ class DataStoreManagerTest extends TestkitExample with SpecificationLike with Mo
       sender.expectMsg(DataManagerResponse(false, "Register Denied. Field Parsing Error: " + "Time field of " + schemaNotATimeField.typeName + "is not in TimeField format.\n"))
       ok
     }
-    "respond success if deregister an existing data model" in {
+  }
+
+  "Data schema deregistering process" should {
+    val sender = new TestProbe(system)
+    val view = new TestProbe(system)
+    val base = new TestProbe(system)
```

Comment:
because they are not created by that name. you created these actors using `childMaker` which ignore the given parameters.